warehouse/query-core/src/main/java/datawave/mr/bulk/MultiRfileInputformat.java

```diff
@@ -152,8 +152,7 @@ public static List<InputSplit> computeSplitPoints(AccumuloClient client, Configu
 
             synchronized (MultiRfileInputformat.class) {
                 final InstanceOperations instOps = client.instanceOperations();
-                dfsUriMap.put(tableId, instOps.getSystemConfiguration().get(Property.INSTANCE_DFS_URI.getKey()));
-                dfsDirMap.put(tableId, instOps.getSystemConfiguration().get(Property.INSTANCE_DFS_DIR.getKey()));
+                dfsUriMap.put(tableId, instOps.getSystemConfiguration().get(Property.INSTANCE_VOLUMES.getKey()));
```

Comment:
So if the dfsDirMap is no longer used, then we should update those other places where we check its contents.